### PR TITLE
fix: Credential of type video wrong id on bookingReference

### LINKED ISSUE #6739: [CAL-941] Zoom meetings don't get removed from zoom app.
While rejecting or cancelling a booking with event type with zoom. Event should be removed from zoom app too.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-941](https://linear.app/calcom/issue/CAL-941/zoom-meetings-dont-get-removed-from-zoom-app)</sub>

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

- Fixes wrong credentialId being saved for bookingReferences, it was always saving credential of type calendar.

Fixes #6739 #8831

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Create an event with location zoom
- [ ] Create a booking for said event
- [ ] Cancel that booking and take look in zoom app if booking was deleted

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/core/EventManager.ts [hunk 1]
```diff
@@ -146,8 +147,8 @@ export default class EventManager {
         meetingId: createdEventObj ? createdEventObj.id : result.createdEvent?.id?.toString(),
         meetingPassword: createdEventObj ? createdEventObj.password : result.createdEvent?.password,
         meetingUrl: createdEventObj ? createdEventObj.onlineMeetingUrl : result.createdEvent?.url,
-        externalCalendarId: evt.destinationCalendar?.externalId,
-        credentialId: evt.destinationCalendar?.credentialId,
+        externalCalendarId: isCalendarType ? evt.destinationCalendar?.externalId : undefined,
+        credentialId: result.credentialId ?? evt.destinationCalendar?.credentialId,
```
Comment (alannnc) [inline packages/core/EventManager.ts:151]: The thing here it was saving calendar Credential ID, when there are some cases that video references need their right value.

**Review by zomars — DISMISSED** [review]

**Review by keithwillcode — DISMISSED** [review]
There are E2E tests failing. Can you please take a look?

File: packages/app-store/dailyvideo/lib/VideoApiAdapter.ts [hunk 2]
```diff
@@ -56,10 +56,10 @@ const meetingTokenSchema = z.object({
 
 /** @deprecated use metadata on index file */
 export const FAKE_DAILY_CREDENTIAL: CredentialPayload & { invalid: boolean } = {
-  id: +new Date().getTime(),
+  id: 0,
   type: "daily_video",
   key: { apikey: process.env.DAILY_API_KEY },
-  userId: +new Date().getTime(),
+  userId: 0,
```
Comment (alannnc) [inline packages/app-store/dailyvideo/lib/VideoApiAdapter.ts:62]: Sometimes credentialId or userId should be saved to DB, but prisma will error if tries to save said values. I think we are better off using 0.
@zomars

**Review by zomars — APPROVED** [review]
Tested and working! Nice job @alannnc 🙏🏽 

Ship it

**Pull request merged**